app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+android:background="@color/white">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Why is the ConstraintLayout nested in a FrameLayout? Can we remove the frame layout?